libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -121,10 +124,24 @@ bool AP_RangeFinder_LightWareSerial::get_reading(float &reading_m)
 
     // all readings were invalid so return out-of-range-high value
     if (invalid_count > 0) {
-        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, max_distance_cm() + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
+        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
         return true;
     }
 
     // no readings so return false
     return false;
 }
+
+// check to see if distance returned by the LiDAR is a known lost-signal distance flag
+bool AP_RangeFinder_LightWareSerial::is_lost_signal_distance(float distance_cm, int16_t distance_cm_max)
+{
+    if (is_equal(130.0f, distance_cm * 0.01f) && distance_cm > distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM) {
+        return true;
+    } else if (is_equal(230.0f, distance_cm * 0.01f) && distance_cm > distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM) {
+        return true;
+    } else if (is_equal(250.0f, distance_cm * 0.01f) && distance_cm > distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Don't repeat code, use early-exit to simplify:
```suggestion
    if (distance_cm < distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM) {
        // in-range
        return false;
    }
    const uint8_8 bad_distances[] { 13000, 23000, 25000 };
    for (const auto bad_distance_cm : bad_distances) {
        if (distance_cm == bad_distance_cm) {
            return true;
        }
    }
    return false;
}
```